test/acceptance/pages.py

```diff
@@ -531,17 +528,6 @@ def verify_available_checked_out_numbers(self, expected_value):
             "Expected avaiable and checked out values present"
         ).fulfill()
 
-    def submissions_available(self):
```

Comment:
Using this method was problematic because the text takes awhile to appear (and thus this method can incorrectly "pass").